pkg/eval/mods/path/path.go

```diff
@@ -176,3 +181,100 @@ func isRegular(path string) bool {
 	fi, err := os.Lstat(path)
 	return err == nil && fi.Mode().IsRegular()
 }
+
+//elvdoc:fn temp-dir
+//
+// ```elvish
+// temp-dir [&dir=$dir] [$pattern]
```

Comment:
Please follow the convention for the signature of othe commands. So this will be:

```
temp-dir &dir=$dir $pattern?
```

Reply:
Changed. Old habits die hard :-)